hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/impl/AWSClientConfig.java

```diff
@@ -32,12 +32,12 @@
 import software.amazon.awssdk.http.apache.ApacheHttpClient;
 import software.amazon.awssdk.http.apache.ProxyConfiguration;
 import software.amazon.awssdk.http.nio.netty.NettyNioAsyncHttpClient;
+import software.amazon.awssdk.thirdparty.org.apache.http.client.utils.URIBuilder;
```

Comment:
this stops people building docker images with more minimal packaging than the bundle.jar; something we had to fix before.